Jenkinsfile

```
@@ -1 +1 @@
-buildPlugin(jdkVersions: [7, 8])
\ No newline at end of file
+buildPlugin(jdkVersions: [8])
```

```suggestion
buildPlugin(configurations: buildPlugin.recommendedConfigurations())
```

Per https://github.com/jenkins-infra/pipeline-library#optional-arguments